FindYourRecipe/FindYourRecipe.Api/Controllers/CategoryController.cs

```diff
@@ -18,6 +16,12 @@ public CategoriesControllers(ICategoryService categoryService)
 			CategoryService = categoryService;
         }
```

Comment:
This paranthesis needs to be moved one tab to the right. For a second I thought here is where the class definition ends. 😄